router/image/templates/nginx.conf

```diff
@@ -39,6 +39,10 @@ http {
     gzip_vary {{ or (getv "/deis/router/gzipVary") "on" }};
     {{ end }}
 
+    # DNS servers to query for address resolution. Needed to resolve OCSP responder.
+    resolver 8.8.8.8 8.8.4.4 valid=300s;
```

Comment:
If `resolver` has to be set for OCSP, maybe its value should be (yet another) etcd key.